HACKING.md

```diff
@@ -14,13 +14,13 @@ Cockpit uses Node.js during development. Node.js is not used at runtime.
 To make changes on Cockpit you'll want to install Node.js, NPM and
 various development dependencies like Webpack.
 
-On Debian or Ubuntu:
+On Debian or Ubuntu, install all the dependencies:
 
-    $ sudo apt-get install nodejs npm
```

Comment:
This is not correct. nodejs is not used at cockpit runtime, thus it's not included in cockpit dependencies.